invenio_users_resources/entity_resolvers.py

```diff
@@ -73,7 +75,12 @@ def pick_resolved_fields(self, identity, resolved_dict):
                 "full_name": profile.get("full_name", ""),
                 "affiliations": profile.get("affiliations", ""),
             },
-            "links": {"avatar": avatar},
+            "links": resolved_dict.get("links"),
+            "blocked_at": resolved_dict.get("blocked_at"),
+            "verified_at": resolved_dict.get("verified_at"),
```

Comment:
to be confirmed if we can expand those fields

Reply:
My hunch is that if we're using `email` as above,  this goes through the service and gets properly filtered, in which case any attribute would be good.

Reply:
true! thanks